Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PropertyBean.java

```diff
@@ -113,33 +113,37 @@ public void onAdapterValueEdit(CellEditEvent event) {
                 new FacesMessage(FacesMessage.SEVERITY_INFO, "INFO", "Property value changed for " + selectedProp.getKey()
                     + " from " + oldValue + " to " + newValue + "."));
         } catch (PropertyAccessException ex) {
-            LOG.warn("Unable to update property: " + selectedProp.getKey());
+            LOG.warn("Unable to update property {}: {}", selectedProp.getKey(), ex.getLocalizedMessage(), ex);
```

Comment:
This was already LOG.warn before this fix; leaving for now, we can reevaluate logging levels for JSF exceptions at another time.